### ISSUE TITLE
Teachable Content Style Guide

### ISSUE THREAD

**Issue by flukeout** [MozillaFoundation/Design#166]
I made some style changes to the Getting Started and Module 1 sections of the [Clubs Training Modules](https://mozilla.teachable.com/courses/mozilla-club-training/lectures/883915) course. Please check it out.

Below are some guidelines that should be applied to the rest of the Modules and other courses.

**Section Headings**
For section headings the _Heading 2_ formatting should be used. When viewing the code for the heading, it should be an `h2` tag.

![headings](https://cloud.githubusercontent.com/assets/25212/14657891/392ac36e-0645-11e6-93c0-050df659fab4.gif)

**Learning Paths**
Learning paths should be regular numbered lists. The time estimate should have normal formatting, but with a bolded title, like this...

![image](https://cloud.githubusercontent.com/assets/25212/14657931/6970b3a8-0645-11e6-8fa7-0e61525a7661.png)

**Text**
- Text should be left-aligned, and not justified.
- To do this, select all the text in the editor box (Command or Control + A) then use the menu in the editor toolbar... 
- For now, let's not apply any custom text colors within the Curriculum Editor
  - We can make global changes later by adding custom CSS in the Teachable editor

![justify](https://cloud.githubusercontent.com/assets/25212/14657953/8835cad0-0645-11e6-9b91-9d4da679bcc6.gif)

**Comment by flukeout** [comment]
cc @amirad @Carotejada

**Comment by amirad** [comment]
Thanks Luke! 

@carotejada can you make these changes to the rest of the module?

**Comment by flukeout** [comment]
@Carotejada if you take a pass at Module 2 I can check it over after and see if there are other recommendations of if my instructions sucked. Thanks!

**Comment by Carotejada** [comment]
They are clear, thank you @flukeout! This is looking better and better.

Will make the changes @amirad

**Comment by xmatthewx** [comment]
This collaborative process is amazing 😍

**Comment by Carotejada** [comment]
@flukeout just finished fixing some of my design sins!  Can you take a look please? 

@xmatthewx it is!

**Comment by flukeout** [comment]
Checking it out now @Carotejada

**Comment by flukeout** [comment]
@Carotejada looks good! One thing I'd recommend is to use the list styling option when you have a list. This way you don't have to add the numbers manually, and later if we want to change the styling of all lists, it will be more predictable.

I've changed the lists to use the list formatting where I've found them in Module 2, like this...

![lists](https://cloud.githubusercontent.com/assets/25212/14691341/8f3474ba-0705-11e6-85fe-2625c99ac9d2.gif)

I'd recommend doing it that way from now on.
